TipCatDotNet.Api/Services/Payments/TransactionService.cs

```diff
@@ -94,6 +103,38 @@ async Task<Result> SetPaymentTime(int memberId)
     }
 
 
+    public async Task<Result<List<FacilityTransactionResponse>>> Get(MemberContext context,
+        TransactionFilterProperty filterProperty, CancellationToken cancellationToken = default)
+    {
+        var facilities = await _context.Facilities
+            .Where(f => f.AccountId == context.AccountId)
+            .ToListAsync(cancellationToken);
+
+        var transactions = await _context.Transactions
+            .GroupBy(t => t.FacilityId)
+            .Select(x => new
+            {
+                Id = x.Key,
+                Total = x.Sum(x => x.Amount),
+                Transactions = x.Skip(Common.DefaultSkip)
+                    .Take(Common.DefaultTop)
+                    .AsQueryable()
+                    .Select(TransactionProjection())
+                    .ToList()
+            })
+            .ToListAsync(cancellationToken);
+
+        var result = new List<FacilityTransactionResponse>(facilities.Count);
+        foreach (var facility in facilities)
+        {
+            var transactionPerFacility = transactions.Where(x => x.Id == facility.Id).FirstOrDefault();
+            result.Add(new FacilityTransactionResponse(facility.Id, facility.Name, transactionPerFacility!.Total, transactionPerFacility!.Transactions));
```

Comment:
Let's move total and transaction count to another endpoint. This is more consistent.